plugins/openai.py

```diff
@@ -50,9 +50,9 @@ def reset_chat_history(cls, view: View, listener: SharedOutputPanelListener, edi
 
     @classmethod
     def create_new_tab(cls, listener: SharedOutputPanelListener):
+        # Everything happens in output_panel.py
         window = active_window()
-        listener.create_new_tab(window)
-        listener.refresh_output_panel(window=window)
+        listener.open_tab(window)   
```

Comment:
The thing is that this feature is quite atomic. Thus you're not allowed to change any code in the outer files (this) completely.